target/linux/pistachio/base-files/etc/init.d/csr

```diff
@@ -0,0 +1,206 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright 2016 by Imagination Technologies Limited and/or its affiliated group companies.
+#
+# All rights reserved.  No part of this software, either
+# material or conceptual may be copied or distributed,
+# transmitted, transcribed, stored in a retrieval system
+# or translated into any human or computer language in any
+# form by any means, electronic, mechanical, manual or
+# other-wise, or disclosed to the third parties without the
+# express written permission of Imagination Technologies
+# Limited, Home Park Estate, Kings Langley, Hertfordshire,
+# WD4 8LZ, U.K.
+
+START=80
+STOP=90
+source /lib/functions/system.sh
+BT_RST_N_GPIO=0
+{
+grep "Marduk" /proc/device-tree/model
```

Comment:
What are thoughts on switching this to use the built in openwrt stuff in lib/functions?